packages/api/cms-api/src/content-generation/content-generation.module.ts

```diff
@@ -31,7 +34,11 @@ export class ContentGenerationModule {
                     provide: CONTENT_GENERATION_SERVICE,
                     useClass: Service,
                 },
-                OpenAiContentGenerationService,
+                {
+                    provide: AZURE_OPEN_AI_CONTENT_GENERATION_SERVICE_CONFIG,
+                    useValue: config.openAiContentGenerationService,
+                },
+                AzureOpenAiContentGenerationService,
```

Comment:
Shouldn't this be a standalone AzureOpenAiContentGenerationModule that provides the config and service?

Reply:
https://github.com/vivid-planet/comet/pull/1712/commits/703ad74ab96dd185dd88b67357cab36c2bce06ae